config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root to: 'users#index'
+  resources :users do
+    resources :posts
```

Comment:
You have exposed some unnecessary routes. Using resources comes with 7 main routes for each model. 

Kindly expose only the index and show routes. See the example below.

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```